### DISCUSSION TITLE
where is the page to download for windows and macos? under current releases its empty?

### DISCUSSION THREAD

**Discussion by bossman762x39** [input-leap/input-leap#1665] [Q&A]
https://github.com/input-leap/input-leap/releases is empty and i cant find a page to download input leap? am i just blind? lol

**Comment by shymega** [comment]
Answer: You can't - we haven't made a release yet.

**Reply by bossman762x39** [reply]
then how are people using it? theres several discussions here and pull requests and activity around input leap but without a way to download it? im coming over from the barrier github and have been using barrier for years and saw this fork and wanted to switch over. has it been ptu to the side? looks like the move over started in 2021.

**Reply by shymega** [reply]
No, it's still active. People use it by using various methods. A release isn't ready yet, and we have been busy - we don't get paid for this project.

**Reply by bossman762x39** [reply]
thank you for the reply, i appreciate what you do and love the project, i do you know if its safe to use barrier 2.3 or 2.4 until a release of input leap?

**Reply by shymega** [reply]
It's best to use Barrier v2.4.0, or at the very least, v2.3.4 - due to security vulns.

If you're on Mac, there's a 'MacPorts Tap' someone made - I need to add that to the wiki - and once we merge the new, updated CI pull request, we'll have Windows installers and macOS DMGs.
